odoo80/Dockerfile

```diff
@@ -25,12 +23,10 @@ RUN apt-get update -q && apt-get upgrade -q \
     postgresql-common \
     libpq-dev \
     libldap2-dev \
-    libsasl2-dev \
     libxml2-dev \
     libxslt1-dev  \
     xsltproc \
     xmlstarlet \
-    openssl \
     python-libxml2 \
     phantomjs \
     bash-completion
```

Comment:
@moylop260 Can you wait until I make the actual pr please? This is WIP as you can read in the tag, I'm testing the changes while I make them to be sure this still works as expected.